src/box/box.h

```diff
@@ -788,6 +788,22 @@ box_iproto_override(uint32_t req_type, iproto_handler_t handler,
 API_EXPORT int64_t
 box_info_lsn(void);
 
+typedef struct slab_info
+{
+	uint64_t items_size;
+	uint64_t items_used;
+	double items_used_ratio;
```

Comment:
IMHO there is no sense in providing `*_ratio` values, anyone can calculate them on the user side.

Reply:
It also may simplify the API if we're going to follow my proposal below to return the values in an array of key-value pairs. If we have only integral values, we don't need to return a type of the value.